recipes/tests.rb

```diff
@@ -381,6 +391,25 @@ module load intelmpi && mpirun --help | grep 'Version 2019 Update 8'
   TESTCUDA
 end
 
+###################
+# FabricManager
+###################
+bash 'test fabric-manager daemon' do
+  cwd Chef::Config[:file_cache_path]
+  code <<-TESTFM
+    nvswitches=$(lspci -d 10de:1af1 | wc -l)
```

Comment:
Could you add a comment in the code about this test? What is `10de:1af1`?

Reply:
can we use the `get_nvswitches` function?